### PR TITLE
Document Lane Splines Tool

### PR DESCRIPTION
**Pull request by plarios**
add documentation for lane spline tool

### CODE REVIEW

File: docs/scenario_editor/roads.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## Creating the Road Network
```
Comment (katiemacaulay) [inline docs/scenario_editor/roads.md:1]: Creating Road Network to create consistency with creating Scenarios

File: docs/scenario_editor/roads.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+### Roads
+The monoDrive Scenario Editor provides a tool for creating road networks that vehicles in the simulation
```
Comment (katiemacaulay) [inline docs/scenario_editor/roads.md:4]: What is the tool's name? Maybe revise to scenario editor provides this _specific_ tool that allows a user to create new road networks.

File: docs/scenario_editor/roads.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+<img class='lg_img' src="https://github.com/monoDriveIO/documentation/raw/lane_tool/WikiPhotos/scenario_editor/roads/monoDrive_road_tool.PNG"/>
+
+Roads are defined by a _reference line_ that represents the basic geometry of the road. Lanes can be added to a road in both the forward and reverse directions, and the lanes follow the path of the reference line. 
+
+A forward lane begins at the location where the road is placed, and ends at the last spline point in the reference line. Similarly, reverse lanes begin at the last spline point and end at the location where the road was placed in the scene.
+
+In the figure below, the blue line is the road's reference line. The blue dots are spline points along the road's reference line.
+
+<img class='lg_img' src="https://github.com/monoDriveIO/documentation/raw/lane_tool/WikiPhotos/scenario_editor/roads/reference_line.PNG"/>
```
Comment (katiemacaulay) [inline docs/scenario_editor/roads.md:17]: Maybe make this image wider to see the blue line. It's hard to see. class='wide_img'

File: docs/scenario_editor/roads.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+<img class='lg_img' src="https://github.com/monoDriveIO/documentation/raw/lane_tool/WikiPhotos/scenario_editor/roads/lane_properties.PNG"/>
+
+Lane segments also have several properties that can be edited as shown in the figure above.
+
+The start and end offset for a lane specify the spline point at which a lane starts or ends. This is useful when creating lane merge scenarios as in the picture below.
+
+<img class='lg_img' src="https://github.com/monoDriveIO/documentation/raw/lane_tool/WikiPhotos/scenario_editor/roads/lane_offsets.PNG"/>
+
+When a road is connected to another road as a successor or a predecessor, all available lanes on the predecessor are connected to those in the successor. This is done from left to right for forward lanes, and from right to left for reverse lanes. Lanes with a start/end offset are not included if the connection point is at the location of the offset.
```
Comment (katiemacaulay) [inline docs/scenario_editor/roads.md:41]: This paragraph was a little confusing to me, had to read it a couple of times to understand. Is there another way or a simpler way to explain this?

File: docs/scenario_editor/roads.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+The monoDrive Scenario Editor provides a tool for creating road networks that vehicles in the simulation
+can drive on. 
+
+To add a road to a map, use the __Actor Place Mode__ in the editor, and search for "monoDrive Road". Then simply drag and drop it into the scene.
+
+<img class='lg_img' src="https://github.com/monoDriveIO/documentation/raw/lane_tool/WikiPhotos/scenario_editor/roads/monoDrive_road_tool.PNG"/>
+
+Roads are defined by a _reference line_ that represents the basic geometry of the road. Lanes can be added to a road in both the forward and reverse directions, and the lanes follow the path of the reference line. 
+
+A forward lane begins at the location where the road is placed, and ends at the last spline point in the reference line. Similarly, reverse lanes begin at the last spline point and end at the location where the road was placed in the scene.
```
Comment (katiemacaulay) [inline docs/scenario_editor/roads.md:13]: Add additional description for forward and reverse just to make it more clear.

**Comment by celite** [comment]
where is the animated gif requested in https://github.com/monoDriveIO/documentation/pull/71#pullrequestreview-390850895

**Pull request merged**